packages/marshal/src/helpers/copyRecord.js

```diff
@@ -52,6 +49,8 @@ export const CopyRecordHelper = harden({
       checkNormalProperty(candidate, name, true, assertChecker);
     }
     // Recursively validate that each member is passable.
-    values(candidate).every(v => !!passStyleOfRecur(v));
+    for (const val of values(candidate)) {
+      passStyleOfRecur(val);
+    }
```

Comment:
Weird that in the same file you replace one old `for` loop with an `.every` and you replace one old `.every` with a `for` loop. Should we instead use the same style in both places?

Reply:
Yeah, I meant to call that out myself. The `.every(…)` sites make use of the return value, while this site just needs to call `passStyleOfRecur` on each value in case that throws an exception. We could arguably make that even more clear here:
```suggestion
    for (const val of values(candidate)) {
      void passStyleOfRecur(val);
    }
```